login-workflow/src/screens/SiteOptionsScreen/types.ts

```diff
@@ -0,0 +1,42 @@
+import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.types';
+import { ErrorManagerProps } from '../../components/Error';
+import { ReactNode } from 'react';
+
+export type SiteOptionsScreenProps = WorkflowCardProps & {
+    /**
+     * The icon to display in the header
+     */
+    icon?: JSX.Element;
+
+    /**
+     * The success message to be displayed on the screen
+     */
+    message?: ReactNode;
+
+    /**
+     * To display label for the next button
+     */
+    nextButtonLabel?: string;
+
+    /**
+     * To display label for the previous button
+     */
+    previousButtonLabel?: string;
```

Comment:
Again, I would use different props such as "joinExistingOrgLabel" and "createNewOrgLabel"